spec/marc_to_solr/lib/princeton_marc_spec.rb

```diff
@@ -744,7 +742,6 @@ def fixture_record(fixture_name)
 
     it 'excludes holdings with an invalid location code' do
       expect(@holdings_with_invalid_location).not_to have_key(@not_valid_holding_id)
```

Comment:
Could we also include a positive expect to make sure that a valid holding id is included?

Reply:
@sandbergja I dont think it is needed because in the `describe '#process_holdings' do` we have different holding_ids to get the holding_block. So it is tested.